aspnetcore/fundamentals/minimal-apis/responses.md

```diff
@@ -164,6 +164,23 @@ app.MapGet("/old-path", () => Results.Redirect("/new-path"));
 app.MapGet("/download", () => Results.File("myfile.text"));
 ```
 
+<a name="httpresultinterfaces7"></a>
+
+### HttpResult interfaces
+
+The following interfaces in the <xref:Microsoft.AspNetCore.Http> namespace provide a way to detect the `IResult` type at runtime, which is a common pattern in filter implementations:
```

Comment:
Add a "For more information see Filters in [Minimal API apps](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/min-api-filters?view=aspnetcore-7.0&branch=pr-en-us-27520)"